### ISSUE TITLE
Order status color selector dropdown does not work correctly with cursor arrow keys

### ISSUE THREAD

**Issue by Saboteur777** [craftcms/commerce#2601]
### Description
Just found a potential a11y issue: order status color selector (on `commerce/settings/orderstatuses/<id>`) dropdown does not work correctly with cursor arrow keys, as it moves two rows instead of one. A screencast is attached. 

### Steps to reproduce

1. Go to `<cpTrigger>/commerce/settings/orderstatuses/`, select an order status to be edited.
2. Open color dropdown and try navigating with up and down cursor arrow keys.

### Additional info

- Craft CMS version: 3.7.19
- Craft Commerce version: 3.4.6
- PHP version: 7.4.11
- Database driver & version: MySQL 5.5.5

https://user-images.githubusercontent.com/9061795/140080228-389e00dd-3c48-46e3-ac9b-7ae84c708ce4.mp4

**Comment by lukeholder** [comment]
Unable to reproduce in Safari and Firefox Developer edition.

Could you confirm your browser. Thanks.

screencast: https://cln.sh/S7CKgJ

**Comment by Saboteur777** [comment]
Created a quick video about it in Firefox 98.0.1 and Chrome 99: https://we.tl/t-Cnlz5oIEtt

The issue is still present, it is happening on Linux, haven't tried it on other platforms.

**Comment by Saboteur777** [comment]
Uploaded some videos: 

https://user-images.githubusercontent.com/9061795/161021403-7169dc8e-d641-4016-b254-b50ddd0f81e5.mp4


https://user-images.githubusercontent.com/9061795/161021411-15f1f449-67c7-462b-9277-09e91024fe83.mp4
